### DISCUSSION TITLE
On the Method of Multi-Point Object Modeling in SionnaRT

### DISCUSSION THREAD

**Discussion by wangfan1005** [NVlabs/sionna#680] [Q&A] [answered]
Hello,

I would like to perform multi-point modeling of an object in SionnaRT. For example, I plan to place a car (object) in the scene, along with a Tx and an Rx, aiming to achieve object sensing through the signal received by the Rx. However, I have a question: Can the car be configured as a target consisting of multiple points in SionnaRT? Perhaps during the generation of reflection paths, a separate path will be calculated for each point?

**Comment by faycalaa** [comment, marked as answer]
Hi @wangfan1005,

Could it be that the ray tracer only finds a single specularly reflected path interacting with the mesh of the car? If so, you could enable diffuse reflection on the car, which should lead to the compute of significantly more paths interacting with it.

**Reply by wangfan1005** [reply]
Hello @faycalaa ,

Thank you for your response. Yes, I can observe a reflected path. I will refer to your answer and conduct further testing. Thank you very much.
